prow/github/client.go

```diff
@@ -114,6 +114,15 @@ type request struct {
 	exitCodes   []int
 }
 
+type requestError struct {
```

Comment:
Are all request errors guaranteed to be this structure?

Reply:
yes, if it can unmarshal the ClientError?